static/js/publisher/release/selectors/index.js

```diff
@@ -386,27 +392,15 @@ export function getSeparatePendingReleases(state) {
           });
         }
 
-        if (changes.length > 0) {
+        if (previousState.key) {
```

Comment:
Now that the key can be `null` this won't cause problems?

Reply:
It actually is more precise - now `changes` contains moving from no progressive release to 100% percentage, whereas `previous.key` will be `null` for all new releases and `{string}` for any updates.